examples/stocks/lib/i18n/stock_messages_all.dart

```diff
@@ -29,7 +29,7 @@ MessageLookupByLibrary _findExact(localeName) {
 }
 
 /// User programs should call this before using [localeName] for messages.
-Future initializeMessages(String localeName) {
+Future<Null> initializeMessages(String localeName) {
```

Comment:
this file is autogenerated so shouldn't be edited (or analyzed)

Reply:
Ok, I wondered why I was getting so many lints in here.  I was just opening directories in Intellij.  :)  @mit-mit gave me instructions on how to do this right now.